.travis.yml

```diff
@@ -8,6 +8,7 @@ php:
   - '7.2'
   - '7.3'
   - '7.4'
+  - 'nightly'
```

Comment:
Remove this one and you will not have 2 builds ;)

Reply:
That seem to have done it. Thank you @williamdes! I guess I was confused here because if you leave it in place but disable all the other matrix builds it only runs one. That's pretty confusing...

Reply:
In fact if you add this one here it will run the matrix job for each value
And you added a special block for a job and that made it to be duplicated.
Here is an example of such override https://github.com/phpmyadmin/sql-parser/blob/37c4baeddee8f749b0ce3a26e88988c5d39dfa60/.travis.yml#L42

Reply:
Right. But I think the current state of the PR is good enough?